capstone-api/routes/feed.js

```diff
@@ -9,22 +9,26 @@ router.get("/news", async (req, res) => {
   let constructedQuery = ""
 
   try {
-    const apiKey = "gJt8QAwXGc_mzm8Nc9Gvj975ZSnKD-A9Pr2oBIkr9g4";
+    const apiKey = "2ynDS7Gsmg-NOYy6PmH8spISzYsfhFCZSqZNU-qAU88";
     const apiUrl = "https://api.newscatcherapi.com/v2/search";
 
     const baseParams = {
         lang: "en", 
         countries: "US", 
         topic: "politics",
-        sources: "foxnews.com,wsj.com,nytimes.com,cnn.com,npr.org,washingtonpost.com,apnews.com,msnbc.com",
         sort_by: "date",
         page_size: 20
     };
 
+    if (loggedIn === "false") {
```

Comment:
Can we move this body of condition to line 28 inside the `else` given this would catch if `Not loggedIn`